pkg/operator/controller.go

```diff
@@ -726,6 +801,59 @@ func (c *Controller) enablePodInformer(enable bool) error {
 	return nil
 }
 
+func (c *Controller) removeResources() error {
+	var lastErr error
+	dsMf := ntomf.TunedDaemonSet()
+
+	_, err := c.listers.DaemonSets.Get(dsMf.Name)
+	if err != nil {
+		if !errors.IsNotFound(err) {
+			lastErr = fmt.Errorf("failed to get DaemonSet %s: %v", dsMf.Name, err)
+		}
+	} else {
+		err = c.clients.Apps.DaemonSets(ntoconfig.OperatorNamespace()).Delete(context.TODO(), dsMf.Name, metav1.DeleteOptions{})
```

Comment:
Do we need unique contexts passed to each delete?

Reply:
I believe [this requirement](https://github.com/kubernetes/client-go/blob/1e6150831a960cf38075bb8561e01f48837f8d74/kubernetes/typed/extensions/v1beta1/daemonset.go#L156) came with the switch to k8s 1.18.
```
# github.com/openshift/cluster-node-tuning-operator/pkg/operator
pkg/operator/controller.go:814:72: not enough arguments in call to c.clients.Apps.DaemonSets("github.com/openshift/cluster-node-tuning-operator/pkg/config".OperatorNamespace()).Delete
        have (string, "k8s.io/apimachinery/pkg/apis/meta/v1".DeleteOptions)
        want (context.Context, string, "k8s.io/apimachinery/pkg/apis/meta/v1".DeleteOptions)
```

Reply:
I was just thinking to have one context variable and just reuse it for each function? `ctx := context.TODO()`?

Reply:
OK, changed, thanks!